compatibility/bazel_tools/testing.bzl

```diff
@@ -855,6 +855,18 @@ excluded_test_tool_tests = [
             },
         ],
     },
```

Comment:
Up at https://github.com/digital-asset/daml/blob/169c332d078a7201c54269bd0338b32ec6b3879e/compatibility/bazel_tools/testing.bzl#L55-L59, we have some comments regarding PRs and exclusions for ContractKeysIT, do we also need to update comments there with our changes on this PR?

Reply:
Good catch. I was not aware of those comments.